6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,22 @@
+// --> Warmup
+
+const args = process.argv.slice(2);
+
+
+if (args.length === 0) {
+    console.log("No arguments provided. Please provide numbers as command line arguments.");
+} else {
+    const numbers = args.map(Number);
+
+    const validNumbers = numbers.filter(num => !isNaN(num));
+
+    if (validNumbers.length > 0) {
+
+        const sum = validNumbers.reduce((acc, num) => acc + num, 0);
+        const average = sum / validNumbers.length;
+
+        console.log("Average:", average);
```

Comment:
One tiny thing for improving string concatenation: 
```suggestion
        console.log(`Average: ${average}`);
```

Reply:
But for the console.log() function, it doesn't really matter :)